### PR TITLE
afpfs-ng: init at 0.8.2

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Things done
- [x] Tested using sandboxing
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using (new package)
- [x] Tested execution of all binary files
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/filesystems/afpfs-ng/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, fuse, readline, libgcrypt, gmp }:
+
+stdenv.mkDerivation rec {
+  name = "afpfs-ng-${version}";
+  version = "0.8.2";
+
+  src = fetchgit {
```
Comment (Mic92) [inline pkgs/tools/filesystems/afpfs-ng/default.nix]: Can `fetchFromGitHub` be used here?

**Review by Mic92 — APPROVED** [review]
otherwise works.

File: pkgs/tools/filesystems/afpfs-ng/default.nix (hunk 1 above)
Comment (rnhmjoj) [inline pkgs/tools/filesystems/afpfs-ng/default.nix]: Yes!

**Comment by Mic92** [comment]
thanks!

**Pull request merged**